core/vm/contracts.go

```diff
@@ -93,7 +93,8 @@ func ecrecoverFunc(in []byte) []byte {
 	vbig := common.Bytes2Big(in[32:64])
 	v := byte(vbig.Uint64())
 
-	if !crypto.ValidateSignatureValues(v, r, s) {
+	// tighter sig s values in homestead only apply to tx sigs
```

Comment:
@fjl there are also tests for them. Try setting the statement to `true` and you'll see the JSON tests fail.